src/test/js/config/test_data.js

```diff
@@ -6,6 +6,6 @@ module.exports = {
     NOTIFY_API_KEY: process.env.NOTIFY_API_KEY,
     SCENARIO_RETRY_LIMIT: 3,
     PASSWORD: "Passw0rdIDAM",
```

Comment:
All secrets should be fetched from keyvault. Do not hardcode.

Reply:
This is not part of my change but still I will check